charts/tfy-gpu-operator/values.yaml

```diff
@@ -139,6 +142,10 @@ aws-eks-gpu-operator:
         - key: nvidia.com/gpu
           effect: NoSchedule
           operator: Exists
+        - key: nvidia.com/fractional-gpu
```

Comment:
Maybe we should probably call this something like `nvidia.com/time-sliced-gpu` to be specific?